src/primitives.jl

```diff
@@ -179,6 +181,30 @@ end
 dropnull(X::NullableVector) = X.values[!X.isnull]    # -> Vector
 
 """
+    dropnull!(X::AbstractVector)
+
+Remove null entries of `X` in-place and return the updated vector. This is a
+no-op if no nulls are present.
+
+    dropnull!(X::NullableVector)
+
+Remove null entries of `X` in-place and return a `Vector` view of `X.values`
```

Comment:
Missing ending dot. Also, I wouldn't mention `X.values` (which is internal), but rather something like "view of its unwrapped entries".